### DISCUSSION TITLE
Blocking external access in a IPv6 network

### DISCUSSION THREAD

**Discussion by milutt** [Hypfer/Valetudo#1328] [Q&A / Support] [answered]
Hi,

Since external access block, I cannot access my S5 via hostname in my local, IPv6 enabled network.
Robot hostname is resolving to both local IPv4 and (public) IPv6 address on my local DNS, requests to robot go over IPv6 and get blocked.
In the logs, I see  following: "Blocked external request to /api/v2/events from <my_laptop_IPv6_address>".

Is there any way to block only IPv4 public addresses without impacting IPv6 traffic?

Thanks

**Comment by rimrul** [comment]
There is no option to block external access only via one of the two. Any IPv6 address that isn't a link-local address, a unique local address or localhost is considered external.

**Comment by schinken** [comment, marked as answer]
Theoratically, a request from the same ipv6 subnet could be interpreted as an internal request.
This is true, as long there is no IPv6 NATing used on the router, of course - so maybe also consider an request from ipv6 default gw as external? 

(Just an idea)

**Reply by schinken** [reply]
Implemented that (except that default gw thingy) here: https://github.com/Hypfer/Valetudo/pull/1342

**Comment by FuhTheGreatest** [comment]
Same issue for me (Deame L10 pro). Seems lika a general thing after the latest update.
